### DISCUSSION TITLE
Add support for the ImageNet Out-of-Distribution (OOD) dataset in MMPreTrain.  支持 ImageNet-A,R,S,C
 Support ImageNet-A, R, S, C in MMPreTrain

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#293] [MMPreTrain]
# Add support for the ImageNet Out-of-Distribution (OOD) dataset in MMPreTrain.  
 支持 ImageNet-A,R,S,C
 Support ImageNet-A, R, S, C in MMPreTrain 

Add support for the ImageNet Out-of-Distribution (OOD) dataset in MMPreTrain. 
 
 Please refer to the pull request [PR]([https://github.com/open-mmlab/mmpretrain/pull/1342)](https://github.com/open-mmlab/mmpretrain/pull/1342))  for guidance.
 1. Complete the code migration.
 2. Adapt the code according to MMPreTrain to ensure correct functionality.
 3. Update the corresponding documentation based on the new structure.
 4. Add the documentation to the appropriate location in index.rst.

给 MMPreTrain 添加 ImageNet OOD 数据集的支持，参考 [PR]([https://github.com/open-mmlab/mmpretrain/pull/1342)](https://github.com/open-mmlab/mmpretrain/pull/1342)) 
 1. 完成代码迁移
 2. 根据 MMPreTrain 进行适配，确保功能实现正确
 3. 根据新的文档结构，添加对应文档
 4. 将文档添加到 index.rst 中对应位置
 

## Task Information

**Repository**：MMPreTrain

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：2023年07月24日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by zeyuanyin** [comment]
PR Link：https://github.com/open-mmlab/mmpretrain/pull/1727
